DataFormats/Common/interface/ValueMap.h

```diff
@@ -14,6 +14,7 @@
 #include <map>
 #include <iterator>
 #include <algorithm>
+#include <stddef.h>
```

Comment:
When I googled where the missing header came from, my first hit was `stddef.h`. Probably using `cstddef` would have been a better choice.